src/Kmd.Logic.Gateway.Automation.Tool/ValidateCommandHandler.cs

```diff
@@ -15,13 +15,20 @@ public async Task<int> Handle(ValidateCommand cmd)
             {
                 using var httpClient = new HttpClient();
                 var gatewayAutomation = new GatewayAutomation(httpClient, this.logicTokenProviderFactory, this.gatewayOptions);
-                var result = await gatewayAutomation.ValidateAsync(cmd.FolderPath).ConfigureAwait(false);
-                foreach (var validationResult in result.ValidationResults)
+                var validationResult = await gatewayAutomation.ValidateAsync(cmd.FolderPath).ConfigureAwait(false);
+                if (validationResult.IsError)
                 {
-                    Console.WriteLine(validationResult.ToString());
+                    foreach (var error in validationResult.Errors)
+                    {
+                        Console.WriteLine(error.ToString());
+                    }
+                }
+                else
+                {
+                    Console.WriteLine(validationResult.ValidatePublishingResult.ToString());
                 }
 
-                return !result.IsError ? 0 : 2;
+                return !validationResult.IsError ? 0 : 2;
```

Comment:
Validation and exception both are error why we have two different error code 1 and 2? May be here we are segregating error type, how we are going to use this?

Reply:
Changed to return value: 1